ios/RNDeviceInfo.xcodeproj/project.xcworkspace/xcshareddata/IDEWorkspaceChecks.plist

```diff
@@ -0,0 +1,8 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
Is this file needed?

Reply:
Actually according to https://developer.apple.com/library/content/releasenotes/DeveloperTools/RN-Xcode/Chapters/Introduction.html

>Xcode 9.3 adds a new IDEWorkspaceChecks.plist file to a workspace's shared data, to store >the state of necessary workspace checks. Committing this file to source control will prevent >unnecessary rerunning of those checks for each user opening the workspace. (37293167)

So I could delete it, but as I understand it is useful file. Should I delete it anyway?

Reply:
Looks harmless and I confirm other high-quality modules have it